CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other
+method with the owner of this repository before making a change.
+
+# Pull Request Process
+1. Get the CoolCompiler code on your local machine
```

Comment:
Please remove *CoolCompiler* and leave it as *Get the code*.

Reply:
Okay I will.